I Added the Info.md hope its fine for now :D (#23)
xf8b/xf8bot: docs/commands/info.md

---
layout: default
title: Info Command
permalink: /docs/commands/info/
---

## Information Command

> The Info command send you a information about xf8bot, GitHub Repository, Documentation and the current prefix 

Reviewer: ```suggestion
The Info command tell you information about xf8bot, such as the GitHub Repository, documentation and the current prefix for this server.
```

just a little touch up

Reviewer 2: tell -> tells

**This is a level one Command.**

> **Usage**: `{prefix}info`

Reviewer: ```suggestion
## Usage
* `{prefix}info`
```

Reviewer: not really needed but felt like it would look more clean this way

GitHub Repository: https://github.com/xf8b/xf8bot/

> Documentation: https://xf8b.github.io/xf8bot/docs/

Reviewer: ```suggestion
Documentation: https://xf8b.github.io/xf8bot/docs/

### **Author: Lone Wolf**
```
